src/agent/src/image_rpc.rs

```diff
@@ -257,72 +164,58 @@ impl ImageService {
                 Err(_) => info!(sl!(), "Attestation Agent already running"),
             }
         }
+        // If the attestation-agent is being used, then enable the authenticated credentials support
```

Comment:
hi @stevenhorsman , here you switch two blocks so that attestation-agent config initilization is done before signature. Does it fix any issue? or it is just your preference?

Reply:
No issue, I just thought that putting all the bits that were based on `aa_kbc_params` was more user friendly.